include/deal.II/fe/fe_tools.h

```diff
@@ -1406,13 +1406,11 @@ namespace FETools
    *
    * @ingroup Exceptions
    */
-  DeclException1(ExcHangingNodesNotAllowed,
-                 int,
-                 << "You are using continuous elements on a grid with "
-                 << "hanging nodes but without providing hanging node "
-                 << "constraints. Use the respective function with "
-                 << "additional AffineConstraints argument(s), instead."
-                 << (arg1 ? "" : ""));
```

Comment:
?? What were we thinking here??

Reply:
I also found that curious. The original commit (720422590b) used `DeclException1` but did not use the integer argument for anything: 3be5dd5886 added something that used the argument to silence a warning.

Believe it or not, we didn't introduce `DeclExceptionMsg` until 847c44d44a in 2015, so the original author used what was available in 2002.